Gemfile

```diff
@@ -21,7 +21,7 @@ gem 'sqlite3', '~> 1.3.6'
 
 # Capybara versions that support RSpec 3 only support RUBY_VERSION >= 1.9.3
 if RUBY_VERSION >= '1.9.3'
-  gem 'capybara', '~> 2.2.0', :require => false
+  gem 'capybara', '~> 2.2', :require => false
```

Comment:
This has a different meaning in semantic versioning that I'm not sure we want, a major release would break this.